lib/AddUserCmd.pm

```diff
@@ -0,0 +1,30 @@
+package AddUserCmd;
+
+use strict;
+use warnings;
+use parent 'Command';
+
+use UserService;
+use Output;
+use Error;
+
+our @ISA = qw(Command);
+
+
+sub excecute {
+
+    my $self = shift;
+    my $user_service = shift || UserService->new();
```

Comment:
the 'shift' command is unnecessary  because the execute() will not send any parameter